package.json

```diff
@@ -64,6 +64,7 @@
     "vitest": "^0.34.6"
   },
   "dependencies": {
+    "@floating-ui/core": "^1.6.1",
```

Comment:
Can you confirm that this is exactly the version that is approved for use in codex? 

Also, let's remove the ^ to make sure we always pull the same version.

Reply:
make sense

codex is using `@floating-ui/vue` 1.0.6, latest version as well, 

we actually need `@floating-ui/dom` here, should we submit another security/performance ticket?

Reply:
It's using the same core component so I think we are ok